src/components/todoDone.js

```diff
@@ -0,0 +1,222 @@
+import React, { useState, useEffect } from "react";
+import styled from "styled-components";
+import TodoListItem from './todoListItem';
+
+const TodoDoneSytle=styled.div`
+    .todoDoneContainer{
+        position:relative;
+        background-color: #262626;
+        border-radius: 8px;
+        width:370px;
+        display:flex;
+        flex-direction:column;
+        flex:1;
+        padding:20px;
+        align-items:center;
+        margin:20px;
+    }
+    .todoDoneTitle{
+        color:white;
+        font-size:37px;
+        text-align: left;
+        width:90%;
+
+    }
+    .todoListPlus{
+        position:absolute;
+        width:26px;
+        height:26px;
+        backgroud-color:white;
+        left:50%;
+        bottom:-13px;
+        border-radius:50%;
+
+    }
```

Comment:
동그란 물체를 바꿀 때 이처럼 px값으로 모두 박아두면 width를 키우고싶을 때 height, bottom모두 바꿔야하는 문제가 있을 수 있어요!
```suggestion
    .todoListPlus{
        position:absolute;
        width:26px;
        aspect-ratio: 1/1;
        backgroud-color:white;
        left:50%;
        bottom:-50%;
        border-radius:50%;
    }
```